testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -2248,4 +2333,47 @@ pub mod test {
             &canonical_stacks_snapshot_is_lower_than_last_winning_snapshot
         ));
     }
+
```

Comment:
Can we start putting unit tests into their own files?  Thanks!

Reply:
I think we need to document a standard for the repo, then refactor accordingly. Mind if we address this in a separate PR, especially since this is just adding one more test to an existing layout?

Reply:
sure